platform/test/spec/testActivityManagerSpec.js

```diff
@@ -0,0 +1,264 @@
+/*global describe, it, expect, spyOn, beforeEach, fail --  functions provided by Jasmine */
+import {ActivityManager} from "../../src/ActivityManager.js";
+
+import { FileHandler } from "../../src/FileHandler.js";
+import { ActivityConfigValidator } from "../../src/ActivityConfigValidator.js";
+import { utility } from "../../src/Utility.js";
+import { ACTIVITY_2PANELS_1ACTION } from "../resources/TestActivityFiles.js";
+import { TOOL_1PANELDEF_1FUNCTION } from "../resources/TestToolFiles.js";
+import { configObjectEquals, configToString } from "../resources/TestUtility.js";
+
+
+describe("ActivityManager", () => {
+
+    it("can be created", () => {
```

Comment:
Given that there are multiple tests for the constructor, does it make sense to group them into a nested `describe` and possibly even reuse the constructed object (with a `beforeAll`)?

Reply:
Fixed in 52769f3d17c78207d5778a9ec46d8e090149bfb0